src/components/schedule-page/validation-drawer/validation-drawer.component.tsx

```diff
@@ -67,3 +68,28 @@ export default function ValidationDrawerComponent(): JSX.Element {
     </div>
   );
 }
+
+export function SpanErrors(props): JSX.Element {
```

Comment:
Brakuje typu dla propsów. Typ jest ważny ponieważ:
1. IDE podpowiada jakie wartości możemy użyć
2. Mamy pewność co do tego co my dostajemy, dzieki czemu możemy uniknąć różnych dziwnych bugów (np. ktoś przekaże zamiast listy funkcję, a ty potem wywolasz na niej `.map`  i poleci `TypeError`)
![obraz](https://user-images.githubusercontent.com/21079319/101178101-dc823c80-3648-11eb-8c6c-9079548cfa34.png)

Reply:
I jeszcze z ciekawości: dlaczego `SpanErrors`?

Reply:
Propsy teraz mogą być typu `ScheduleErrorMessageModel[] | undefined`.

A `SpanErrors` dlatego że definiuję linijkę (span) z przyciskami i napisem :smile:

Reply:
Moim zdaniem w takim razie bardziej pasuje `ErrorDrawerHeader` czy coś w tym stylu. Chodzi o to, żeby kończyło się na `Header`